lib/elixir/test/elixir/stream_test.exs

```diff
@@ -925,50 +925,49 @@ defmodule StreamTest do
            [a: {:tea, 2}, c: {:coffee, 1}]
   end
 
-  test "zip/2" do
+  test "zip/1" do
```

Comment:
I wasn't sure if it was ok to replace these specs, but `zip/2` is now just a pass through to this verison.